client/src/types/global-cloudflare.d.ts

```diff
@@ -0,0 +1,2 @@
+declare const XATA_DATABASE_URL: string | undefined;
+declare const XATA_API_KEY: string | undefined;
```

Comment:
[question] Probably rename to something like ``global-variables.d.ts`` as other vendors other than CF could use the same approach?